src/LightPanels/LightPanels.js

```diff
@@ -805,7 +809,8 @@ module.exports = kind(
 		this._currentPanel = nextPanel;
 
 		// ensure that `transitionFinished` is called in the case where we are not animating
-		if (!this.shouldAnimate() || !animate) this.transitionFinished(null, null, true);
+		if (!this.shouldAnimate() || !animate) this.transitionFinished();
```

Comment:
Good question. :smile: I had actually wavered on this and split this into a `cleanUpTransition` method and a `transitionFinished` handler as it *looked* cleaner. I ultimately scrapped this in favor of the current solution, to not add another method, and to reduce the surface area of this change. If this is a pattern we want to follow, I am definitely not opposed to making that change, but maybe it makes more sense in a next-gen version?